src/js/main.js

```diff
@@ -122,6 +120,26 @@ var SupportKit = Marionette.Object.extend({
             .done();
     },
 
+    logout: function() {
+        // delete the deviceid cookie
+        document.cookie = SK_STORAGE + '=; expires=Thu, 01 Jan 1970 00:00:01 GMT;';
```

Comment:
Hmm.. this is nice, but how does it logs out the user? Unless `destroy` is called before that, the widget is still fully operational. Collections are still initialized and the headers are set up to use the previous id. Also, is the JWT per user or per app? If per user, it needs to be cleared too. 

Shouldn't we just bundle or alias `logout` and `destroy` and add the cookie clearing logic in the `onDestroy` handler?